src/gravity/universe.py

```diff
@@ -0,0 +1,21 @@
+import pygame as pg
+from planet import Planet
+from constants import *
+
+
+class Universe:
+    def __init__(self):
+        self.planets = [
+            Planet(10, 10, (WIDTH / 2 - 100, HEIGHT / 2), (0, 50)),
+            Planet(10, 10, (WIDTH / 2 + 100, HEIGHT / 2), (0, -50)),
+        ]
+
+    def update(self, dt):
+        for planet in self.planets:
+            planet.calcUpdate(self.planets)
+        for planet in self.planets:
+            planet.applyUpdate(dt)
```

Comment:
notice I am using calcUpdate first to calculate the strength of gravitational field due to other planets. and then I applyUpdate to change positions of all planets to gether

Reply:
Oh yeah, multiple for loops does sound like a better idea. 👍